### PR TITLE
Sle15 fix

### PR DESCRIPTION
**Pull request by marcosps**

### CODE REVIEW

File: files/salt/openSUSE-15.0-x86_64_tiny/etc/sysconfig/clock [hunk 1]
```diff
@@ -0,0 +1,2 @@
+TIMEZONE="America/New_York"
```
Comment (jschmid1) [inline files/salt/openSUSE-15.0-x86_64_tiny/etc/sysconfig/clock]: This timezone is not the best default for all users and should be kept local

File: files/salt/openSUSE-15.0-x86_64_tiny/etc/ntp.conf [hunk 2]
```diff
@@ -0,0 +1,154 @@
+
```
Comment (jschmid1) [inline files/salt/openSUSE-15.0-x86_64_tiny/etc/ntp.conf]: This file provided within the salt/admin/etc/ntp.conf directory and should end up on the node anyways

File: files/salt/openSUSE-15.0-x86_64_tiny/root/.bash_history [hunk 3]
```diff
@@ -0,0 +1,36 @@
+salt-run state.orch ceph.stage.0
```
Comment (jschmid1) [inline files/salt/openSUSE-15.0-x86_64_tiny/root/.bash_history]: If you didn't change something, this file should end up on the node anyways. You don't need to specify this explicitly.

File: files/salt/openSUSE-15.0-x86_64_tiny/root/.vimrc [hunk 4]
```diff
@@ -0,0 +1,185 @@
+set nocompatible              " be iMproved, required
```
Comment (jschmid1) [inline files/salt/openSUSE-15.0-x86_64_tiny/root/.vimrc]: same here

**Comment by marcosps** [comment]
@jschmid1 by pushing this PR automatically merges #35 ?

Please let me know if these patches need improvements.

**Pull request merged**